src/compiler/checker.ts

```diff
@@ -14520,15 +14517,13 @@ namespace ts {
                     if (hasProperty(node.locals, local)) {
                         const localValue = node.locals[local];
                         if (localValue.declarations && !localValue.exportSymbol) {
-                            for (const declaration of localValue.declarations) {
-                                const symbol = declaration.symbol;
-                                if (!symbol.hasReference) {
-                                    if (declaration.kind === SyntaxKind.ImportSpecifier || declaration.kind === SyntaxKind.ImportClause || declaration.kind === SyntaxKind.NamespaceImport) {
-                                        error(declaration, Diagnostics._0_is_declared_but_never_used, symbol.name);
-                                    }
-                                    else if (declaration.kind === SyntaxKind.ImportEqualsDeclaration) {
-                                        error(declaration.name, Diagnostics._0_is_declared_but_never_used, symbol.name);
-                                    }
+                            const declaration = localValue.declarations[0];
+                            if (!localValue.hasReference) {
+                                if (declaration.kind === SyntaxKind.ImportSpecifier || declaration.kind === SyntaxKind.ImportClause || declaration.kind === SyntaxKind.NamespaceImport) {

```

Comment:
i am still not sure i understand why checkUnusedImports is not part of the checkUnusedModuleLocals?